src/components/VisualTesterKeymap.vue

```diff
@@ -32,14 +32,14 @@
           {{ displayKeyCode }}
         </div>
       </div>
-      <textarea
+      <div
         class="status-log"
         id="terminal"
         ref="status"
-        cols="120"
-        rows="5"
-        v-model="status"
-      />
+        spellcheck="false"
+        style="width:869px;text-align:left;"
```

Comment:
let's extract this to <style>

Reply:
this needs some refactor of css stylesheet because it uses also the `#terminal` like homepage.